File: app/src/main/java/com/example/routecraft/data/pojos/Address.java
package com.example.routecraft.data.pojos;

import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity()
public class Address {

    @PrimaryKey(autoGenerate = true)
    private int id;
    private String address;

    public Address(String address) {
        this.address = address;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }
}

File: app/src/main/java/com/example/routecraft/data/pojos/Address.kt
package com.example.routecraft.data.pojos

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity
data class Address(
        @PrimaryKey
        val addressId: Int,
        val address: String = "",
        val street: String,
        val postCode: String = "",
        val city: String,
        val country: String = "",
        val lat: Double = 0.0,
        val lng: Double = 0.0
)